app/views/messages/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<div class="container">
+  <div class="title">Messenger</div>
+  <div class="window">
+    <div class="text" id="chat-output"></div>
+  </div>
+</div>
+<div class="container">
+  <div class="chatbox">
+    <%= form_for Message.new, remote: true do |msg| %>
```

Comment:
make this in a variable in index action
`@message_new = current_company.message.new`